go-controller/pkg/ovn/controller/network_qos/network_qos.go

```diff
@@ -325,6 +325,10 @@ func (c *Controller) networkManagedByMe(networkSelectors crdtypes.NetworkSelecto
 		switch networkSelector.NetworkSelectionType {
 		case crdtypes.DefaultNetwork:
 			return c.IsDefault(), nil
+		case crdtypes.PrimaryUserDefinedNetworks:
+			return c.IsPrimaryNetwork(), nil
```

Comment:
should we further check `networkSelector.PrimaryUserDefinedNetworkSelector.NamespaceSelector`?

Reply:
ah, yes. There is no `NetworkSelector` in `PrimaryUserDefinedNetworkSelector` but we definitely could make this smarter. PTAL.